shapely/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "1.8.5.post1"
+__version__ = "1.8.5.post1.post1"
```

Comment:
Note that this look strange but is valid and done rather than "1.8.5.post2" in case upstream actually releases that version

Reply:
nvm this actually may be invalid as the `packaging` library doesn't recognize it

Reply:
going back to `.post2`